packages/core/src/external/carequality/utils.ts

```diff
@@ -34,3 +34,51 @@ export function cleanXml(xml: string): string {
   xml = xml.replace(/\\"/g, '"');
   return xml;
 }
+
+interface ParsedContentType {
+  boundary: string;
+  startToken: string;
+}
+
+function parseContentTypeHeader(header: string): ParsedContentType | undefined {
+  console.log("header", header);
+  const boundaryMatch = header.match(/boundary="([^"]+)"/);
+  const startTokenMatch = header.match(/start="<([^"]+)>"/);
```

Comment:
The header props might not be between double quotes - see [example](https://www.ibm.com/docs/en/cics-ts/5.3?topic=data-mtomxop-soap).

This brings a question: couldn't we be relying on an existing implementation/library to work with SOAP? Do we have to implement this?

Example lib, with support to MTOM: https://www.npmjs.com/package/soap